### PR TITLE
Critical metrics for SCSI disks added

### PR DESCRIPTION
**Pull request by wwwlde**
Added the following metrics for SCSI disks:

- Grown Defect List
- Read Errors Corrected by ReReads/ReWrites
- Read Total Uncorrected Errors
- Write Errors Corrected by ReReads/ReWrites
- Write Total Uncorrected Errors

### CODE REVIEW

**Comment by k0ste** [comment]
@wwwlde can you rebase this PR?
Thanks!

**Comment by Nexucis** [comment]
@wwwlde do you want to finish this PR.
It looks like it is waited by many users, would be cool to see this one merge !

**Comment by wwwlde** [comment]
@Nexucis Done!

**Review by Nexucis — COMMENTED** [review]
it looks ok to me, but I don't have much knowledge around this topic.
@SuperQ could look at this PR and say if you have anything against merging it ?

**Comment by SuperQ** [comment]
Yikes, the rebase created a merge mess.

File: smartctl.go [hunk 1]
```diff
@@ -435,3 +436,60 @@ func (smart *SMARTctl) mineDeviceERC() {
...
+			prometheus.CounterValue,
+			SCSIHealth.Get("read.errors_corrected_by_rereads_rewrites").Float(),
+			smart.device.device,
+			smart.device.family,
+			smart.device.model,
+			smart.device.serial,
+		)
+		smart.ch <- prometheus.MustNewConstMetric(
+			metricReadTotalUncorrectedErrors,
+			prometheus.CounterValue,
```
Comment (SuperQ) [inline smartctl.go:469]: This either needs to be a `GaugeValue`, or the metric name needs to end in `_total`.

Same goes for all the other metrics.

**Review by SuperQ — CHANGES_REQUESTED** [review]

**Comment by k0ste** [comment]
@wwwlde now this merged to master and can be closed
Thanks!

**Comment by Nexucis** [comment]
closing this PR in favor of #131 

Thanks for your help here @wwwlde !

**Pull request closed without merging**